docs/content/docs/user-docs/openshift.md

```diff
@@ -70,6 +70,7 @@ ACK_ENABLE_DEVELOPMENT_LOGGING=true
 ACK_LOG_LEVEL=debug
 ACK_WATCH_NAMESPACE=
 AWS_REGION=us-west-2
+AWS_ENDPOINT_URL=
 ACK_RESOURCE_TAGS=hellofromocp
```

Comment:
This is a funny placeholder value but it might be good to have a standardised default so we can track OCP usage more easily

Reply:
@RedbackThomson Totally open to another value, just wasn't sure what it should be.